yarn-project/ethereum/src/contracts/rollup.ts

```diff
@@ -177,19 +188,44 @@ export class RollupContract {
     return this.rollup.read.getCurrentSlot();
   }
 
-  async getCommitteeAt(timestamp: bigint) {
-    const { result } = await this.client.simulateContract({
-      address: this.address,
+  async getCommitteeAt(timestamp: bigint): Promise<readonly `0x${string}`[]> {
+    const { result } = await this.l1TxUtils.simulate(
+      {
+        to: this.address,
+        data: encodeFunctionData({
+          abi: RollupAbi,
+          functionName: 'getCurrentCommittee',
+        }),
+      },
+      {
+        // @note we add 1n to the timestamp because geth implementation doesn't like simulation timestamp to be equal to the current block timestamp
+        time: timestamp + 1n,
```

Comment:
Why is the + 1 only required on this function call and not applied to all others?

Reply:
good point, I should probably apply it everywhere